utils/generate-commands-json.py

```diff
@@ -29,57 +31,64 @@ def convert_keyspec(spec):
     return spec
 
 
-def convert_entry_to_objects_array(container, cmd):
+def convert_entry_to_objects_array(container, cmd, details):
     """Transform the JSON output of `COMMAND` to a friendlier format.
 
-    `COMMAND`'s output per command is a fixed-size (8) list as follows:
+    cmd is the output of `COMMAND` as follows:
     1. Name (lower case, e.g. "lolwut")
```

Comment:
This looks like markdown, so... blank line before list. :)

```suggestion
    cmd is the output of `COMMAND` as follows:

    1. Name (lower case, e.g. "lolwut")
```

Reply:
looks like, but it's not. just a block of text..
i rather not waste excessive lines on this comment, and it'll never be viewed with a tool that'll mess it up